Orange/widgets/utils/colorpalette.py

```diff
@@ -597,66 +597,52 @@ def __getitem__(self, val):
 
 
 class ColorPaletteGenerator:
-    maxHueVal = 260
 
-    def __init__(self, number_of_colors=None, rgb_colors=DefaultRGBColors):
+    def __init__(self, number_of_colors=0, colors=DefaultRGBColors):
```

Comment:
Changing the name of a constructor parameter for a class that is (in)directly used in most graphical widgets usually requires some refactoring.
I am guessing you didn't use the Find Usages feature of PyCharm (or grep) or tried some standard Orange widgets after the changes? At least the Scatter Plot and Data Table widgets crash and probably some others as well...